contracts/src/v0.8/ChainSpecificUtil.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import {ArbSys} from "./vendor/@arbitrum/nitro-contracts/src/precompiles/ArbSys.sol";
+
+//@dev A library that abstracts out opcodes that behave differently across chains.
+//@dev The methods below return values that are pertinent to the given chain.
+//@dev For instance, ChainSpecificUtil.getBlockNumber() returns L2 block number in L2 chains
+library ChainSpecificUtil {
+    address private constant ARBSYS_ADDR =
+        address(0x0000000000000000000000000000000000000064);
+    ArbSys private constant ARBSYS = ArbSys(ARBSYS_ADDR);
+    uint256 private constant ARB_MAINNET_CHAIN_ID = 42161;
+    uint256 private constant ARB_GOERLI_TESTNET_CHAIN_ID = 421613;
```

Comment:
How are we going to handle e.g an Arbitrum Sepolia (assuming this is inevitable with Goerli being deprecated)?